blocks/to-do/add-list-items.js

```diff
@@ -6,17 +6,22 @@ export default function addListItem(inputValue, list, completed) {
 
   // Create elements
   const listItem = document.createElement('li');
+  const listItemCheckbox = document.createElement('input');
   const listItemBtn = document.createElement('button');
 
   // Add class and attribute
   listItem.classList.add('to-do__list-item');
   listItem.setAttribute('data-completed', completed);
+  listItemCheckbox.setAttribute('type', 'checkbox');
+
+  listItemCheckbox.checked = completed;
 
   // Change content
   listItem.textContent = inputValue;
   listItemBtn.innerHTML = '&#10006;';
 
   // Append children
+  listItem.appendChild(listItemCheckbox);
```

Comment:
this is an accessibility violation. if you add an input element, you should either add a `label` for that element (or wrap the text of the todo in that label) or add an `aria-label` metadata to the input element.

note: in this case, since you have a perfect label already displayed in the UI, an `aria-label` is actually counter-productive in terms of accessibility, since assisted reading technology would just announce the checkbox twice if you added that metadata...

Reply:
and of course if you change your `li` content to be `label` elements, make sure to check the events and css to make sure nothing's broken.